packages/core/src/cart/buy-now-cart-request-body.ts

```diff
@@ -1,3 +1,5 @@
+import { CartSource } from './cart-source';
```

Comment:
would be better to use it from [packages/payment-integration-api/src/cart/cart-source.ts](https://github.com/bigcommerce/checkout-sdk-js/pull/1773/files#diff-173aaeb1e68e6b3807e1ce6e2da6072fc33a3b528581af0cbb1b5dd30dadc796)

Reply:
in all other cases in `core` package as Mykhailo said